src/app/map/map.component.scss

```diff
@@ -1,22 +1,48 @@
+@import "../../scss/common";
+
 .map-container {
-  $height: 600px;
-  s4e-view-manager {
-    display: block;
-    width: 300px;
-    height: $height;
-    float: left;
-  }
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  z-index: 10;
+  width: 289px;
+  border: 1px solid #333333;
+  background-color: #ffffff;
+  color: $gray-900;
+}
 
-  #map {
-    display: block;
-    width: 600px;
-    height: $height;
-    float: left;
-  }
+#map {
+  display: block;
+  width: 100%;
+  height: 100vh;
+  float: left;
 }
 
 .product-views {
   li {
     cursor: hand;
   }
 }
+
+.timeline {
+  bottom: 20px;
+  left: 400px;
+  z-index: 100;
+}
+
+.ol-zoom {
```

Comment:
This should go to the styles.scss file. When included here it doesn't have effect.